codelab/03_buildpacks-runimage-override/app/go.mod

```diff
@@ -1,3 +1,3 @@
-module github.com/GoogleContainerTools/skaffold/examples/buildpacks
+module github.com/GoogleContainerTools/skaffold/codelabs/buildpacks
```

Comment:
should this be singular `codelab` rather than `codelabs`? To match the directory name?